.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: li11mQP9V16CrLOrtGsWWn42f3LuqDr9C
```

Comment:
:see_no_evil: Token and other authentication resources shouldn't be pushed to remote :speak_no_evil: